src/ConnectedVMware/ConnectedVMware.Autorest/docs/New-AzConnectedVMwareVM.md

```diff
@@ -0,0 +1,930 @@
+---
+external help file:
+Module Name: Az.ConnectedVMware
+online version: https://docs.microsoft.com/powershell/module/az.connectedvmware/new-azconnectedvmwarevm
+schema: 2.0.0
+---
+
+# New-AzConnectedVMwareVM
+
+## SYNOPSIS
+The operation to Create a virtual machine instance.
+Please note some properties can be set only during virtual machine instance creation.
+
+## SYNTAX
+
+### CreateExpanded (Default)
+```
+New-AzConnectedVMwareVM -MachineId <String> [-ExtendedLocationName <String>] [-ExtendedLocationType <String>]
+ [-HardwareProfileMemorySizeMb <Int32>] [-HardwareProfileNumCoresPerSocket <Int32>]
+ [-HardwareProfileNumCpUs <Int32>] [-InfrastructureProfileFirmwareType <String>]
```

Comment:
For this one mayve we can suppress the error, but do you think it's better to be `[-HardwareProfileNumCPUs <Int32>]`?

Reply:
existing on more readable, we can keep that.

Reply:
Yes, we can suppress the `should be singular` error, but it is expected to be `[-HardwareProfileNumCpus <Int32>]` rather than `[-HardwareProfileNumCpUs <Int32>]`